internal/storage/plan9/plan9.go

```diff
@@ -14,6 +15,8 @@ import (
 	"golang.org/x/sys/unix"
 )
 
+const PacketPayloadBytes = 65536
```

Comment:
NIT: can we not export this since its not used outside the package?

Reply:
Fixed.